docs/pages/development.js

```diff
@@ -243,6 +243,63 @@ yarn cypress open
 ~~~"
                   />
                 </li>
+                <li>
+                  <Text>
+                    Run{' '}
+                    <Link href="https://github.com/meinaart/cypress-plugin-snapshots" inline>
+                      <Text weight="bold">Cypress snapshot tests</Text>
+                    </Link>
+                    . If any component changes are expected to visually modify your component, you
+                    must update the snapshot tests. In order to run the tests, you must start the
+                    documentation server in one terminal and the Cypress testing interface in
+                    another.
+                  </Text>
+                  <Markdown
+                    text="
+~~~bash
+yarn start
+~~~"
+                  />
+                  <Markdown
+                    text="
+~~~bash
+yarn cypress open
+~~~"
+                  />
+                  <Flex direction="column">
+                    <Box as="figure" width={400}>
+                      <Image
+                        alt=""
+                        color="white"
+                        naturalHeight={262}
+                        naturalWidth={400}
+                        src="https://i.ibb.co/FY1pp4Y/Screen-Shot-2022-03-09-at-4-49-21-PM.png"
+                      />
+                      <Text size="100" align="center">
+                        <Box as="figcaption" marginTop={3}>
+                          {`Each failing snapshot test with display an error message. Click on
+                          "Compare Snapshot". It will open a separate interface displaying the
+                          expected and the received snapshots side by side for your comparison.`}
+                        </Box>
+                      </Text>
+                    </Box>
+                    <Box as="figure" width={400}>
+                      <Image
+                        alt=""
+                        color="white"
+                        naturalHeight={176}
+                        naturalWidth={400}
+                        src="https://i.ibb.co/7NyhPRD/Screen-Shot-2022-03-09-at-4-49-40-PM.png"
+                      />
+                      <Text size="100" align="center">
+                        <Box as="figcaption" marginTop={3}>
+                          {`On this new interface displaying the expected and the received snapshots,
+                          click on the "Update snapshot" button to reconcile the changes.`}
```

Comment:
This should include language about updating only if the changes are expected, otherwise to revisit the code

Reply:
DONE